.github/workflows/native_s3_dlr.yml

```diff
@@ -0,0 +1,34 @@
+name: Native S3 DLR
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        operating-system: [macos-latest]
+    env:
+      DLR_VERSION: "1.5.0"
```

Comment:
I don't like have two place to store version.
I think we should only store version in gradle.properties file.
1. Use gradlew to invoke build.sh file and pass version to the shell
2. in the shell script create a folder with version name, we just copy the parent folder to s3 without need  to know the version

Reply:
agreed though it is native build not jni build. I use bash tool to get the version from gradle property